shared/mpapiResponse.ts

```diff
@@ -54,3 +54,7 @@ export function determineAppStore(subscription: AppSubscription) {
 export function isPuzzle(subscription: AppSubscription) {
 	return subscription.productId.includes('puzzles');
 }
+
+export function isFeast(subscription: AppSubscription) {
+	return subscription.productId.includes('Feast');
```

Comment:
I've found a live productId with a lowercase 'f' so I would suggest making the check case insensitive.